### ISSUE TITLE
Storing data in a cross-language form

### ISSUE THREAD

**Issue by martinfleis** [Nowosad/spData#71]
Hi,

would you be keen on storing the data in some open formats alongside rda so we could link to it from Python? We have a `geodatasets` package that holds metadata and some tooling to cache the data locally so if you including the data here as GeoJSON, CSV, GPKG or whatever is needed we could include them in `geodatasets` allowing easier access to the same data from R and Python, avoiding the need of running R first to save the data Python can read.

**Comment by Robinlovelace** [comment]
Would be really useful to have cross-language datasets. Maybe a spDatapy or spDatax repo could be worthwhile, to avoid issues with CRAN..

**Comment by Nowosad** [comment]
@martinfleis what do you have in mind? Do you want to store the files in some python package? Many of the datasets from spData are available in inst/shapes -- https://github.com/Nowosad/spData/tree/master/inst/shapes (although we plan to remove shapefiles soon from there -- https://github.com/Nowosad/spData/issues/62). Do you need any other dataset from spData as a file?

**Comment by martinfleis** [comment]
> Many of the datasets from spData are available in inst/shapes

Missed that! That is what I was looking for. If these links are considered stable, I would just include them in `geodatasets` for easy access from Python.

**Comment by Nowosad** [comment]
Yes, they are v. stable. (Except the .shp files, which will be removed in ~two months)

**Comment by Nowosad** [comment]
@martinfleis .gal files are at https://github.com/Nowosad/spData/tree/master/inst/weights

**Comment by martinfleis** [comment]
I have exposed those datasets that live in `inst/shapes` in geodatasets in https://github.com/geopandas/geodatasets/pull/27. It is far from the complete list but I believe that the rest is not available as files but generated in some form?

**Comment by Nowosad** [comment]
The rest of them are .rda object -- do you want all of the datasets from the README available (except the one we discussed yesterday)? If so, I could just create another GH repo for that.

**Comment by Robinlovelace** [comment]
+1 to increasing modularity and x-language compat (without having to depend on either for shared examples).

**Comment by Nowosad** [comment]
@martinfleis I took a look at the data available in R files -- they consist of spatial vector data, some raster data, a few tables, and also some graph data. Do you have any suggestions on the data formats you would prefer for each of the data types (e.g., vector -- gpkg, raster -- geotiff, etc)?

**Comment by martinfleis** [comment]
As long as GDAL can read it I don't really care.

**Comment by Nowosad** [comment]
@martinfleis take a look at https://github.com/Nowosad/spData_files and let me know what you think. 

@rsbivand Roger, do you maybe also have any suggestions on how to store and share these datasets?

**Comment by martinfleis** [comment]
I think it will work for me if we don't touch it (so the shasum won't change).

**Comment by rsbivand** [comment]
I agree that using file formats that have active GDAL drivers is sensible; for larger data sets maybe SOzip from when GDAL has provided that? Otherwise for vector zipped files may reduce bandwidth: https://github.com/r-spatial/sf/issues/2433 and https://github.com/Nowosad/spData/issues/62#issuecomment-2378946906.

**Comment by Nowosad** [comment]
Thanks, Roger. 

@martinfleis -- what do you think? Should I compress the gpkg files or keep them as they are?

**Comment by martinfleis** [comment]
@Nowosad given the largest file in that repo is 905KB, I don't think compression is worth it.

**Comment by rsbivand** [comment]
Maybe also https://github.com/Nowosad/spDataLarge/tree/master/inst  and the `RDA` files in https://github.com/Nowosad/spDataLarge/tree/master/data?

**Comment by Nowosad** [comment]
Good idea, @rsbivand -- I just added some additional files from spData and all of the files from spDataLarge to https://github.com/Nowosad/spData_files

**Comment by martinfleis** [comment]
This all looks fine with me. I'll wait a bit if there are any further comments and then expose these in `geodatasets`.
